testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/SuiteClients.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.launcher.suite.suites;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tests.product.launcher.env.EnvironmentConfig;
+import io.trino.tests.product.launcher.env.environment.EnvMultinode;
+import io.trino.tests.product.launcher.suite.Suite;
+import io.trino.tests.product.launcher.suite.SuiteTestRun;
+
+import java.util.List;
+
+import static io.trino.tests.product.launcher.suite.SuiteTestRun.testOnEnvironment;
+
+public class SuiteClients
+        extends Suite
+{
+    @Override
+    public List<SuiteTestRun> getTestRuns(EnvironmentConfig config)
+    {
+        return ImmutableList.of(
+                testOnEnvironment(EnvMultinode.class)
+                        .withGroups(
+                                "cli",
+                                "jdbc",
+                                "trino_jdbc")
+                        .withExcludedGroups("ldap", "ldap_and_file")
```

Comment:
Why exclude these groups? It's not "splitting up tests" if (new Suite1 + SuiteClients + SuiteFunctions + SuiteTpch) != old Suite1

Reply:
Suite 1 excludes the `PROFILE_SPECIFIC_TESTS` group, which includes all LDAP CLI tests. Note that profile means environment. I thought excluding them explicitly is better, but that group can include more tests specific to other environments, not a generic one like `multinode`. I changed the exclusion here.